api/v1alpha1/inventory_types.go

```diff
@@ -71,38 +75,43 @@ type AlarmSubscriptionServerConfig struct {
 	ServerConfig `json:",inline"`
 }
 
-// ORANO2IMSSpec defines the desired state of ORANO2IMS
-type ORANO2IMSSpec struct {
-	// INSERT ADDITIONAL SPEC FIELDS - desired state of cluster
-	// Important: Run "make" to regenerate code after modifying this file
-
+// InventorySpec defines the desired state of Inventory
+type InventorySpec struct {
 	// Image is the full reference of the container image that contains the binary. This is
 	// optional and the default will be the value passed to the `--image` command line flag of
 	// the controller manager.
-	//
 	//+optional
+	//+operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Image",xDescriptors={"urn:alm:descriptor:com.tectonic.ui:text"}
 	Image string `json:"image"`
 	// CloudId is used to correlate the SMO inventory record with the deployed cloud instance.
+	//+operator-sdk:csv:customresourcedefinitions:type=spec,displayName="CloudId",xDescriptors={"urn:alm:descriptor:com.tectonic.ui:text"}
 	CloudId string `json:"cloudId"`
 	//+optional
+	//+operator-sdk:csv:customresourcedefinitions:type=spec,displayName="MetadataServerConfig",xDescriptors={"urn:alm:descriptor:com.tectonic.ui:text"}
 	MetadataServerConfig MetadataServerConfig `json:"metadataServerConfig"`
 	//+optional
+	//+operator-sdk:csv:customresourcedefinitions:type=spec,displayName="DeploymentManagerServerConfig",xDescriptors={"urn:alm:descriptor:com.tectonic.ui:text"}
 	DeploymentManagerServerConfig DeploymentManagerServerConfig `json:"deploymentManagerServerConfig"`
+	// ResourceServerConfig contains the configuration for the resource server.
 	//+optional
+	//+operator-sdk:csv:customresourcedefinitions:type=spec,displayName="ResourceServerConfig",xDescriptors={"urn:alm:descriptor:com.tectonic.ui:text"}
 	ResourceServerConfig ResourceServerConfig `json:"resourceServerConfig,omitempty"`
+	// AlarmSubscriptionServerConfig contains the configuration for the alarm server.
 	//+optional
+	//+operator-sdk:csv:customresourcedefinitions:type=spec,displayName="AlarmSubscriptionServerConfig",xDescriptors={"urn:alm:descriptor:com.tectonic.ui:text"}
 	AlarmSubscriptionServerConfig AlarmSubscriptionServerConfig `json:"alarmSubscriptionServerConfig"`
 	// IngressHost defines the FQDN for the IMS endpoints.
 	//+optional
+	//+operator-sdk:csv:customresourcedefinitions:type=spec,displayName="IngressHost",xDescriptors={"urn:alm:descriptor:com.tectonic.ui:text"}
 	IngressHost string `json:"ingressHost,omitempty"`
 }
 
 type DeploymentsStatus struct {
-	// +operator-sdk:csv:customresourcedefinitions:type=status,displayName="Deployment Server Status"
+	// +operator-sdk:csv:customresourcedefinitions:type=status,displayName="DeploymentServerStatus"
```

Comment:
Shouldn't the spaces in the `displayName` be preserved?   Isn't this used in UI elements for end users?

Reply:
I used [TALM](https://github.com/openshift-kni/cluster-group-upgrades-operator/blob/main/pkg/api/clustergroupupgrades/v1alpha1/types.go) as an example which uses both ways, so I've decided to go with no spaces. I can change all of them to include spaces.